src/main/java/io/weaviate/client/v1/misc/model/MultiVectorConfig.java

```diff
@@ -17,6 +18,8 @@ public class MultiVectorConfig {
   private boolean enabled = true;
   @Builder.Default
   private Aggregation aggregation = Aggregation.MAX_SIM;
+  @SerializedName("muvera")
+  private MuveraConfig encoding;
```

Comment:
```suggestion
  private MuveraConfig muvera;
```
As other types of encoding might be added later, let's not "use up" this variable name. Like in the example with quantizers, the fields are called `bq`/`pq` and not `quantization`:

https://github.com/weaviate/java-client/blob/e1b39e6fa764419f1549a2b0eb7f04d7bba6cee6/src/main/java/io/weaviate/client/v1/misc/model/VectorIndexConfig.java#L30-L32

Leaving explicit `@SerializedName` annotation is good form.

Now, in order have the builder expose `encoding(...)` method and not `.muvera(...)`, you can define one of your own:

```java
public class MultiVectorConfig {
  // ...
  public static MultiVectorConfigBuilder {
    private MuveraConfig muvera;

    // Promote this method
    public MultiVectorConfigBuilder encoding(MuveraConfig muvera) {
      this.muvera = muvera;
      return this;
    }
    
    // Hide the default setter implementation
    private MultiVectorConfigBuilder muvera(MuveraConfig _muvera) {
      return this;
    }

  }
}
```

Lombok will merge this definition of `MultiVectorConfigBuilder` with the one it generates, so the public API will stay unchanged.

Reply:
To be clear, we _could_ also leave that as is and only introduce that change later when we have a different encoding configuration without breaking the API.

What will change in that case is the name of the getter. Instead of `getEncoding()` there should be `getMuveraEncoding`. Having overloaded `encoding(Muvera)` and `encoding(Other)` is fine, but we can't have the same for getters.